Jenkinsfile

```diff
@@ -40,32 +39,16 @@ timestamps() {
                 echo("===================================")
             }
 
-            stage('compile with Eclipse Luna and Xtext 2.9.2') {
-                sh "${mvnHome}/bin/mvn -P!tests -Declipsesign=true -Dtarget-platform=eclipse_4_4_2-xtext_2_9_2 ${mvnParams} clean install"
+            stage('compile with Eclipse 2023-03 and Xtext 2.31.0') {
+                sh "${mvnHome}/bin/mvn -P!tests -Declipsesign=true -Dtarget-platform=eclipse_2023_03-xtext_2_31_0 ${mvnParams} clean install"
                 archiveArtifacts artifacts: 'org.eclipse.xpect.releng/p2-repository/target/repository/**/*.*,org.eclipse.xpect.releng/p2-repository/target/org.eclipse.xpect.repository-*.zip'
             }
 
             wrap([$class: 'Xvnc', useXauthority: true]) {
 
-                stage('test with Eclipse Luna and Xtext 2.9.2') {
+                stage('test with Eclipse 2023-06 and Xtext nighly') {
                     try{
-                        sh "${mvnHome}/bin/mvn -P!plugins -P!xtext-examples -Dtarget-platform=eclipse_4_4_2-xtext_2_9_2 ${mvnParams} clean integration-test"
-                    }finally{
-                        junit '**/TEST-*.xml'
-                    }
-                }
-
-                stage('test with Eclipse Mars and Xtext 2.14') {
-                    try{
-                        sh "${mvnHome}/bin/mvn -P!plugins -P!xtext-examples -Dtarget-platform=eclipse_4_5_0-xtext_2_14_0 ${mvnParams} clean integration-test"
-                    }finally {
-                        junit '**/TEST-*.xml'
-                    }
-                }
-
-                stage('test with Eclipse 2022-03 and Xtext nighly') {
-                    try{
-                        sh "JAVA_HOME=${java11Home} ${mvnHome}/bin/mvn -P!plugins -P!xtext-examples -Dtycho-version=2.7.5 -Dtarget-platform=eclipse_2022_03-xtext_nightly ${mvnParams} clean integration-test"
+                        sh "JAVA_HOME=${java11Home} ${mvnHome}/bin/mvn -P!plugins -P!xtext-examples -Dtycho-version=2.7.5 -Dtarget-platform=eclipse_2023_06-xtext_nightly ${mvnParams} clean integration-test"
```

Comment:
this does not seem to work.
202306 will need java 17 anyway

Reply:
Oh yes, this cannot work. Can we test the Jenkinsfile of this PR (after I have fixed it)? Jenkins builds this PR with the Jenkinsfile from master which fails (due to Java 8 etc.).